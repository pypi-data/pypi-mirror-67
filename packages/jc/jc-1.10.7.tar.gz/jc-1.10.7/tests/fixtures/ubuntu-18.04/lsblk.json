[{"name": "fd0", "maj_min": "2:0", "rm": true, "size": "1.4M", "ro": false, "type": "disk", "mountpoint": null}, {"name": "loop0", "maj_min": "7:0", "rm": false, "size": "54.5M", "ro": true, "type": "loop", "mountpoint": "/snap/core18/1223"}, {"name": "loop1", "maj_min": "7:1", "rm": false, "size": "11M", "ro": true, "type": "loop", "mountpoint": "/snap/slcli/383"}, {"name": "loop2", "maj_min": "7:2", "rm": false, "size": "88.7M", "ro": true, "type": "loop", "mountpoint": "/snap/core/7396"}, {"name": "loop3", "maj_min": "7:3", "rm": false, "size": "66.5M", "ro": true, "type": "loop", "mountpoint": "/snap/google-cloud-sdk/103"}, {"name": "loop4", "maj_min": "7:4", "rm": false, "size": "66.5M", "ro": true, "type": "loop", "mountpoint": "/snap/google-cloud-sdk/104"}, {"name": "loop5", "maj_min": "7:5", "rm": false, "size": "54.4M", "ro": true, "type": "loop", "mountpoint": "/snap/core18/1074"}, {"name": "loop7", "maj_min": "7:7", "rm": false, "size": "8.6M", "ro": true, "type": "loop", "mountpoint": "/snap/doctl/187"}, {"name": "loop8", "maj_min": "7:8", "rm": false, "size": "3.1M", "ro": true, "type": "loop", "mountpoint": "/snap/stress-ng/847"}, {"name": "loop9", "maj_min": "7:9", "rm": false, "size": "8.6M", "ro": true, "type": "loop", "mountpoint": "/snap/doctl/215"}, {"name": "loop10", "maj_min": "7:10", "rm": false, "size": "89.1M", "ro": true, "type": "loop", "mountpoint": "/snap/core/7917"}, {"name": "loop11", "maj_min": "7:11", "rm": false, "size": "3.2M", "ro": true, "type": "loop", "mountpoint": "/snap/stress-ng/924"}, {"name": "sda", "maj_min": "8:0", "rm": false, "size": "20G", "ro": false, "type": "disk", "mountpoint": null}, {"name": "sda1", "maj_min": "8:1", "rm": false, "size": "1M", "ro": false, "type": "part", "mountpoint": null}, {"name": "sda2", "maj_min": "8:2", "rm": false, "size": "20G", "ro": false, "type": "part", "mountpoint": "/"}, {"name": "sr0", "maj_min": "11:0", "rm": true, "size": "64.8M", "ro": false, "type": "rom", "mountpoint": null}, {"name": "sr1", "maj_min": "11:1", "rm": true, "size": "848M", "ro": false, "type": "rom", "mountpoint": null}]
