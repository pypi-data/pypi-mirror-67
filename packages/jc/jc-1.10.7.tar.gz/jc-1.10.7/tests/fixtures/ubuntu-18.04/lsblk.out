NAME   MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
fd0      2:0    1  1.4M  0 disk 
loop0    7:0    0 54.5M  1 loop /snap/core18/1223
loop1    7:1    0   11M  1 loop /snap/slcli/383
loop2    7:2    0 88.7M  1 loop /snap/core/7396
loop3    7:3    0 66.5M  1 loop /snap/google-cloud-sdk/103
loop4    7:4    0 66.5M  1 loop /snap/google-cloud-sdk/104
loop5    7:5    0 54.4M  1 loop /snap/core18/1074
loop7    7:7    0  8.6M  1 loop /snap/doctl/187
loop8    7:8    0  3.1M  1 loop /snap/stress-ng/847
loop9    7:9    0  8.6M  1 loop /snap/doctl/215
loop10   7:10   0 89.1M  1 loop /snap/core/7917
loop11   7:11   0  3.2M  1 loop /snap/stress-ng/924
sda      8:0    0   20G  0 disk 
├─sda1   8:1    0    1M  0 part 
└─sda2   8:2    0   20G  0 part /
sr0     11:0    1 64.8M  0 rom  
sr1     11:1    1  848M  0 rom  
